pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -70,6 +70,10 @@ func parseConfig(config *zconfig.EdgeDevConfig, getconfigCtx *getconfigContext,
 		// Network objects are used for systemAdapters
 		networksChanged := parseNetworkXObjectConfig(config, getconfigCtx)
 		if physioChanged || networksChanged {
+			// system adapter configuration that we send to zedrouter, depends
```

Comment:
Can you change this if block to:

forceParse := physioChanged || networksChanged
parseSystemAdapterConfig(config, getconfigCtx, forceParse)

Much more readable. You can retain the comment..